packages/cli/lib/lib/webpack/render-html-plugin.js

```diff
@@ -148,9 +150,9 @@ module.exports = async function(config) {
 // Adds a preact_prerender_data in every folder so that the data could be fetched separately.
 class PrerenderDataExtractPlugin {
 	constructor(page) {
-		const { url } = page.CLI_DATA.preRenderData;
+		const { url } = page.UNSAFE_CLI_DATA.preRenderData;
```

Comment:
can we `deserialize` here? so that there is only one data source.

Reply:
This is already addressed